guides/source/configuring.md

```diff
@@ -440,7 +440,7 @@ The schema dumper adds two additional configuration options:
 
 * `config.action_controller.per_form_csrf_tokens` configures whether CSRF tokens are only valid for the method/action they were generated for.
 
-* `config.action_controller.default_protect_from_forgery` determines whether forgery protection is added on `ActionController::Base`. This is false by default.
+* `config.action_controller.default_protect_from_forgery` determines whether forgery protection is added on `ActionController::Base`. For new applications, the default is true.
```

Comment:
We can follow other places and simply say:

> This is true by default.

We change a lot of these options, and in this guide we should just mention what the defaults are.